lms/djangoapps/teams/models.py

```diff
@@ -48,9 +49,11 @@ def create(cls, name, course_id, description, topic_id=None, country=None, langu
         """
 
         team_id = generate_unique_readable_id(name, cls.objects.all(), 'team_id')
+        discussion_id = 'team_id:' + team_id
```

Comment:
@jimabramson - I'd like to get your thoughts on this. I was initially concerned about this, but @benmcmorran pointed out that discussion_ids being set by the LMS is a pattern we've seen elsewhere. Given that we know that team_ids are unique within teams, and by namespacing them to `'team_id:'` we prevent collisions unless an instructor chooses a really odd discussion ID for a forum topic.